cdap-client-tests/src/test/java/co/cask/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -598,15 +598,18 @@ public void testSystemMetadataRetrieval() throws Exception {
     // verify stream system metadata
     StreamId streamId = NamespaceId.DEFAULT.stream(AllProgramsApp.STREAM_NAME);
     Set<String> streamSystemTags = getTags(streamId, MetadataScope.SYSTEM);
-    Assert.assertEquals(ImmutableSet.of(AllProgramsApp.STREAM_NAME, AbstractSystemMetadataWriter.EXPLORE_TAG),
+    Assert.assertEquals(ImmutableSet.of(AbstractSystemMetadataWriter.EXPLORE_TAG),
                         streamSystemTags);
 
     Map<String, String> streamSystemProperties = getProperties(streamId, MetadataScope.SYSTEM);
     // Verify create time exists, and is within the past hour
-    final String creationTime = "creation-time";
-    String description = "description";
-    String schema = "schema";
-    String ttl = "ttl";
+    String creationTime = AbstractSystemMetadataWriter.CREATION_TIME;
```

Comment:
why copy to a local String variable before using them? Why notj ust use the static variables directly?